common/models/user.js

```diff
@@ -580,7 +580,10 @@ module.exports = function(User) {
             query && query.redirect !== undefined ? query.redirect :
             undefined;
 
-          ctx.res.redirect(redirectUrl);
+          if (redirectUrl !== undefined) {
+            ctx.res.location(redirectUrl);
```

Comment:
Per my comment in https://github.com/strongloop/strong-remoting/pull/174, this code should set both response location and status code.